app/abilities/about-alpha-video.js

```diff
@@ -0,0 +1,7 @@
+import { inject as service } from '@ember/service';
+import { Ability } from 'ember-can';
+
+export default Ability.extend({
```

Comment:
There is no about-alpha-videos page right? Then we don't need a ability I think?

Reply:
Okay then I'll remove that. Don't know enough about Ember yet so I just copy pasted some stuff 😅